project/BuildSettings.scala

```diff
@@ -0,0 +1,127 @@
+import sbt.Keys._
+import sbt._
+
+object BasicSettings extends AutoPlugin {
+  override def trigger = allRequirements
+
+  override def projectSettings = Seq(
+    organization := "com.mohiva",
+    version := "2.0-SNAPSHOT",
+    resolvers ++= Dependencies.resolvers,
```

Comment:
@rfranco The resolver `"Atlassian Releases" at "https://maven.atlassian.com/public/"` must be the first because of this bug: https://groups.google.com/forum/#!topic/sbt-dev/EQVZGcYtBp0